packages/yoroi-extension/chrome/extension/ergo-connector/api.js

```diff
@@ -374,7 +375,7 @@ export async function connectorSignTx(
       dataBoxesToSpend,
     );
   debug('signedTx', '', signedTx);
-  return signedTx.to_json();
+  return JSONBigInt.parse(signedTx.to_json());
```

Comment:
Same thing here, check the function return type: `ErgoTxJson`

Output values and output asset amounts must be strings:

![image](https://user-images.githubusercontent.com/5585355/133500954-f8a50c5d-7833-4149-9838-ddd13181e443.png)

So gotta do explicit conversion